.travis.yml

```diff
@@ -8,22 +8,9 @@ python:
   - 3.5
 
 env:
-  - TOX_ENV=django3
-  - TOX_ENV=django22
-  - TOX_ENV=django21
   - TOX_ENV=django20
   - TOX_ENV=django111
 
-matrix:
-  include:
-    - python: 2.7
-      env: TOX_ENV=django111
-    - python: pypy
-      env: TOX_ENV=django111
-  exclude:
-    - python: 3.5
-      env: TOX_ENV=django3
```

Comment:
If you keep this exclusion entry, could you also keep the `django3` TOX_ENV above?